pkg/features/pilot/pilot.go

```diff
@@ -78,6 +78,13 @@ var (
 	// File based certificates are located under $BaseDir/etc/certs/. If not set, the original 1.0 locations will
 	// be used, "/"
 	BaseDir = "BASE"
+
+	// HTTP10 enables the use of HTTP10 in the outbound HTTP listeners, to support legacy applications.
+	// Will add "accept_http_10" to http outbound listeners. Can also be set only for specific sidecars via meta.
+	//
+	// Alpha in 1.1, may become the default or be turned into a Sidecar API or mesh setting. Only applies to namespaces
+	// where Sidecar is enabled.
+	HTTP10 = os.Getenv("PILOT_HTTP10") == "1"
```

Comment:
We should test with 
`os.Getenv("PILOT_HTTP10") != "0"` that way we know that all tests pass even when this is enabled by default.

It is expected to be a compatible option.

I am sure actual 1.0 calls were tested elsewhere.

Reply:
Not to say that we need to turn this on by default "now".

Reply:
We did test it in the 'perf' cluster - I have a pod with the option turned on. John is working on an automated test for this ( using the metrics ), but will be in the tools repo.

Reply:
Automation PR in tools repo: https://github.com/istio/tools/pull/50